.travis.yml

```diff
@@ -2,37 +2,26 @@ language: php
 
 sudo: false
 
-php:
-  - 7.0
-  - 7.1
-
-env:
-  - SYMFONY_VERSION=~2.8.0
-  - SYMFONY_VERSION=~3.0
-  - SYMFONY_VERSION=~4.0
-  - DEPENDENCIES=beta
-
 matrix:
   fast_finish: true
-  exclude:
-    - php: 5.6
-      env: DEPENDENCIES=beta
-    - php: 7.0
-      env: SYMFONY_VERSION=~4.0
-    - php: 7.0
-      env: DEPENDENCIES=beta
+  include:
+    - php: 7.2
+      env: SYMFONY_VERSION=~2.8.0
+    - php: 7.3
+      env: SYMFONY_VERSION=^3.0
+    - php: 7.3
```

Comment:
Not testing on 7.4 as there are some incompatibilities with phpunit. I will update these dependencies in another PR.